docs/product-user-guide/src/main/asciidoc/decision-tables-defining-proc.adoc

```diff
@@ -0,0 +1,51 @@
+[id='decision-tables-defining-proc']
+= Defining decision tables
+
+Decision table spreadsheets (XLS or XLSX) require two key areas that define rule data: a `RuleSet` area and a `RuleTable` area. The `RuleSet` area of the spreadsheet defines elements that you want to apply globally to all rules in the same package (not only the spreadsheet), such as a rule set name or universal rule attributes. The `RuleTable` area defines the actual rules (rows) and the conditions, actions, and other rule attributes (columns) that constitute that rule table within the specified rule set. A decision table spreadsheet can contain multiple `RuleTable` areas, but only one `RuleSet` area.
+
+IMPORTANT: You should create only one decision table spreadsheet, containing all necessary `RuleTable` definitions, per rule package in {CENTRAL}. You can create separate decision table spreadsheets for separate packages, but multiple decision table spreadsheets in the same package can cause compilation errors from conflicting `RuleSet` or `RuleTable` attributes.
+
+*<@Michael (1 of 5): I've added the above note both here and in the "Uploading decision tables" section to clarify that a single spreadsheet should be used. That work?>*
```

Comment:
@sterobin A single workbook per package (RuleSet) is encouraged (for most Users) but not essential (if you know what you're doing). Therefore IDK whether "IMPORTANT: You should create only one...." is too strong wording vs "IMPORTANT: You are encouraged to create only one...". Final decision with you!

Reply:
Got it. Will consider slight rewording.